### DISCUSSION TITLE
添加语言支持根据ip所在地配置用户语言支持全球语言

### DISCUSSION THREAD

**Discussion by zhanhailin123** [lsky-org/lsky-pro#788] [想法]
添加语言支持
根据ip所在地配置用户语言
支持全球语言切换
根据用户所在地的语言配置 第一次设置的语言  或者能能自己设置默认的语言为其他国家语言

**Comment by FreeyW** [comment]
的确很需要多语言支持，最起码要添加全英语的支持
